docs/ipsec-tunnel.md

```diff
@@ -1,15 +1,18 @@
 # IPsec Encryption of Tunnel Traffic with Antrea
 
-Antrea supports encrypting tunnel traffic across Nodes with IPsec ESP. At this
-moment, IPsec encyption works only for GRE tunnel (but not Geneve, VXLAN, and
-STT tunnel types).
+Antrea supports encrypting tunnel traffic across Nodes with IPsec ESP.
+IPsec encyption works for all OVS supported tunnel including Geneve, GRE,
+VXLAN, and STT tunnel.
 
 ## Prerequisites
 
 IPsec requires a set of Linux kernel modules. Check the required kernel modules
 listed in the [strongSwan documentation](https://wiki.strongswan.org/projects/strongswan/wiki/KernelModules).
 Make sure the required kernel modules are loaded on the Kubernetes Nodes before
 deploying Antrea with IPsec encyption enabled.
+If you want to enable IPsec with Geneve, please make sure [this commit](https://github.com/torvalds/linux/commit/34beb21594519ce64a55a498c2fe7d567bc1ca20) is included
+in your kernel. For Ubuntu 18.04, it should be a version after `4.15.0-124`. For Ubuntu 20.04, it should be a version after `5.4.67`.
+If you'd like to enable IPSec with STT, please be ware that the `vport-stt` module is not in the Linux tree and needs to be built from source, please build and load it manually before IPSec with STT tunneling is enabled.
```

Comment:
I feel no need to mention the STT prerequisite here. It is documented in getting-started already.

Reply:
removed